### PR TITLE
[ASRangeController] Remove beta indicator at class names

### PR DESCRIPTION
**Pull request by maicki**
The ASRangeController was recently replaced with a new implementation. For the transition period we added a beta indicator to the ASRangeController classes at the end. This Pull Request will remove the beta indicators and furthermore clean up some code path's that are not necessary anymore due to the transition to the new ASRangeController.

Open questions / tasks:
- [x] Remove ASRangeHandler, ASRangeHandlerRender, ASRangeHandlerPreload, ASRangeHandlerVisible

### CODE REVIEW

File: AsyncDisplayKit/Details/ASLayoutController.h [hunk 1]
```diff
@@ -31,10 +31,6 @@ FOUNDATION_EXPORT BOOL ASRangeTuningParametersEqualToRangeTuningParameters(ASRan
 
 - (ASRangeTuningParameters)tuningParametersForRangeMode:(ASLayoutRangeMode)rangeMode rangeType:(ASLayoutRangeType)rangeType;
 
-// FIXME: This method can be removed once ASRangeControllerBeta becomes the main version.
-// TODO: Now that it is the main version, can we remove this now?
-- (BOOL)shouldUpdateForVisibleIndexPaths:(NSArray<NSIndexPath *> *)indexPaths rangeType:(ASLayoutRangeType)rangeType;
-
```
Comment (nguyenhuy) [inline AsyncDisplayKit/Details/ASLayoutController.h]: :dancer: :dancers:

**Comment by nguyenhuy** [comment]
Awesome, thanks for working on this :) I think we no longer need `ASRangeHandlerRender`.

**Comment by appleguy** [comment]
@maicki excited to land this as soon it builds! Indeed, we can't entirely delete all forms of ASRangeHandler, including the base class

**Comment by maicki** [comment]
@appleguy Build errors should be resolved now

Maybe we can even further refactor a bit. If I look into ASDisplayNode.mm there is the comment: "Once the working window is eliminated (ASRangeHandlerRender), trigger display directly here.". As we don't use ASRangeHandlerRender anymore we should be able to trigger the display directly now and remove the if / else.

**Comment by appleguy** [comment]
@maicki this diff is happiness!  Thank you!

**Pull request merged**